sim/pkg/model/replica_entity.go

```diff
@@ -112,18 +121,23 @@ func (re *replicaEntity) GetCPUCapacity() float64 {
 	return re.totalCPUCapacityMillisPerSecond
 }
 
+func (re *replicaEntity) GetCreationTimeStamp() time.Time {
+	return re.creationTimeStamp
+}
+
 func NewReplicaEntity(env simulator.Environment, failedSink *simulator.SinkStock) ReplicaEntity {
 	replicaNum++
 
 	re := &replicaEntity{
 		env:                                env,
 		number:                             replicaNum,
-		totalCPUCapacityMillisPerSecond:    100,
+		totalCPUCapacityMillisPerSecond:    1000,
 		occupiedCPUCapacityMillisPerSecond: 0,
 	}
 
+	re.creationTimeStamp = re.env.CurrentMovementTime()
 	re.requestsComplete = simulator.NewSinkStock(simulator.StockName(fmt.Sprintf("RequestsComplete [%d]", re.number)), "Request")
 	re.requestsProcessing = NewRequestsProcessingStock(env, re.number, re.requestsComplete, failedSink, &re.totalCPUCapacityMillisPerSecond, &re.occupiedCPUCapacityMillisPerSecond)
-
+	re.tickTock = NewMetricsTickTockStock(env, re)
```

Comment:
You're doing something slightly unexpected here, in that every replica has its own tick tock.  We previously discussed a simpler model in which there was one tick tock (just like the autoscaler) which collects metrics from all replicas.

But this is actually a little better because the metric freshness of each replica will vary slightly, depending on when it (and its tick tock) were started.  This is what happens in real life, because each kubelet is scraping at different intervals (at least with docker runtime).  With containerd, I believe that metrics are taken at that instant.  But there will still be variation in the Metric Server scraping time.

Anyway, this is fine.